packages/api/amplify_api_dart/pubspec.yaml

```diff
@@ -0,0 +1,27 @@
+name: amplify_api_dart
+description: The Amplify API category plugin in Dart-only, supporting GraphQL and REST operations.
+version: 0.0.1
+homepage: https://docs.amplify.aws/lib/q/platform/flutter/
+repository: https://github.com/aws-amplify/amplify-flutter/tree/next/packages/api/amplify_api_dart
+issue_tracker: https://github.com/aws-amplify/amplify-flutter/issues
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+
+dependencies:
+  amplify_core: ">=1.0.0-next.2 <1.0.0-next.3"
+  async: ^2.8.0
+  aws_common: ">=0.3.3 <0.4.0"
+  collection: ^1.15.0
+  json_annotation: ^4.6.0
+  meta: ^1.7.0
+  plugin_platform_interface: ^2.0.0
+  stream_transform: ^2.0.1
+  web_socket_channel: ^2.2.0
+
+dev_dependencies:
+  amplify_lints: 
+    path: ../../amplify_lints
+  aws_signature_v4:
+  stream_channel: ^2.1.0
```

Comment:
Add three more dev deps and re-run `aft generate workflows`. This will make sure both web targets are tested as well.

```yaml
  build_runner: ^2.0.0
  build_test: ^2.1.5
  build_web_compilers: ^3.2.4
```